internal/monitors/elasticsearch/metadata.yaml

```diff
@@ -24,76 +24,115 @@ monitors:
+
+    ```yaml
     monitors:
     - type: elasticsearch
-     host: localhost
-     port: 9200
-     enableIndexStatsPrimaries: true
+      host: localhost
+      port: 9200
+      threadPools:
+      - bulk
+      - warmer
+      - listener
     ```
 
-    It is possible to collect index level stats that are aggregated across all indexes, rather than one a per index level
+    Here is a list of valid thread pools by Elasticsearch version:
 
-    ```
+    | thread pool name | ES 1.x | ES 2.0 | ES 2.1+ |
+    |------------------|--------|--------|--------|
+    | merge            | &#x2713;      |        |        |
+    | optimize         |&#x2713;     |        |        |
+    | bulk             |&#x2713;     |&#x2713;     |&#x2713;     |
+    | flush            |&#x2713;     |&#x2713;     |&#x2713;     |
+    | generic          |&#x2713;     |&#x2713;     |&#x2713;     |
+    | get              |&#x2713;     |&#x2713;     |&#x2713;     |
+    | snapshot         |&#x2713;     |&#x2713;     |&#x2713;     |
+    | warmer           |&#x2713;     |&#x2713;     |&#x2713;     |
+    | refresh          |&#x2713;     |&#x2713;     |&#x2713;     |
+    | fetch\_shard\_started|      |&#x2713;     |&#x2713;     |
+    | fetch\_shard\_store|        |&#x2713;     |&#x2713;     |
+    | listener         |        |&#x2713;     |&#x2713;     |
+    | management       |        |&#x2713;     |&#x2713;     |
+    | percolate        |        |&#x2713;     |&#x2713;     |
+    | suggest          |        |&#x2713;     |&#x2713;     |
+    | force\_merge      |        |        |&#x2713;     |
+
+
+    ## Collecting index statistics
+
+    By default, the configuration parameter `indexes` is empty, which means
+    collect stats on all indexes. To collect statistics from a subset of
+    indexes, set the configuration parameter `indexes` to a list of the index
+    names you want to collect stats for.
+
+    The call to collect index statistics can be CPU-intensive. For this reason
+    SignalFx recommends using the `indexStatsIntervalSeconds` configuration
+    parameter to decrease the reporting interval for nodes that report index
+    statistics.
+
+    ### Primaries vs total
+    By default the monitor collects a subset of index stats of total aggregation
+    type (see docs for details). It is possible to enable index stats of primaries
+    aggregation type too. Total for an index stat aggregates across all shards.
+    Whereas, Primaries only reflect the stats from primary shards. An example
+    configuration to enable index stats from Primary shards too:
+
+    ```yaml
     monitors:
     - type: elasticsearch
-     host: localhost
-     port: 9200
-     IndexSummaryOnly: true
+      host: localhost
+      port: 9200
+      enableIndexStatsPrimaries: true
     ```
+
+    ## Built-in content
+
+    For more information on the built-in content we have for Elasticsearch,
+    [see
+    here](https://github.com/signalfx/integrations/tree/master/elasticsearch)
```

Comment:
This link doesn't work. Do you mean this? https://github.com/signalfx/integrations/tree/master/collectd-elasticsearch   Or maybe the repo docs are in the process of being renamed/re-organized, and this link will work when this PR is merged?

But also, maybe link to docs instead?  https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations.elasticsearch.html

Reply:
This related PR (https://github.com/signalfx/integrations/pull/686) will move the old `collectd-elasticsearch` dir to just `elasticsearch` and then the link should work.